packages/teleport/src/teleportContext.ts

```diff
@@ -70,4 +70,8 @@ export default class Context implements teleport.Context {
   isTrustedClustersEnabled() {
     return this.storeUser.getTrustedClusterAccess().list;
   }
+
+  isUsersViewEnabled() {
```

Comment:
`isUsersEnabled`  (implying that this is a feature)

Reply:
revised